config/application.rb

```diff
@@ -31,5 +31,7 @@ class ApplicationLetter < Rails::Application
 
     config.from_address = ENV['SMTP_ADDRESS'] ? ENV['SMTP_ADDRESS'] : 'noreply@localhost'
     config.reply_to_address = config.from_address
+
+    config.empty_agreement_letter_path = File.read(Rails.root.join('storage', 'agreement_letters', 'real_agreement_letter.pdf'))
```

Comment:
I think you should remove the path suffix, because it Really is the content, not the path.